rpm_spec/subpackages/manageiq-core

```diff
@@ -36,6 +36,17 @@ done
 %files core
 %defattr(-,root,root,775)
 %{app_root}
+%{_prefix}/lib/systemd/system/manageiq-cockpit_ws*
+%{_prefix}/lib/systemd/system/manageiq-ems_metrics_processor*
+%{_prefix}/lib/systemd/system/manageiq-event_handler*
+%{_prefix}/lib/systemd/system/manageiq-generic*
+%{_prefix}/lib/systemd/system/manageiq-priority*
+%{_prefix}/lib/systemd/system/manageiq-remote_console*
+%{_prefix}/lib/systemd/system/manageiq-reporting*
+%{_prefix}/lib/systemd/system/manageiq-schedule*
+%{_prefix}/lib/systemd/system/manageiq-smart_proxy*
+%{_prefix}/lib/systemd/system/manageiq-ui*
+%{_prefix}/lib/systemd/system/manageiq-web_service*
```

Comment:
Not sure if I can do a regex like `%{_prefix}/lib/systemd/system/manageiq-(cockpit_ws|event_handler|...)*`

Reply:
Looks like I could build up a file during the `%install` section like `find -regex ... > core_systemd_files` and use that later in the `%files` section.

Up to you guys if you prefer that over this enumerated list